frontend/components/Header/Nav/EditionDropdown.tsx

```diff
@@ -22,32 +23,68 @@ const editionDropdown = css`
     }
 `;
 
-const EditionDropdown: React.SFC = () => {
-    const links: Link[] = [
-        {
-            url: '/preference/edition/uk',
-            title: 'UK edition',
-            isActive: true,
-        },
-        {
-            url: '/preference/edition/us',
-            title: 'US edition',
-        },
-        {
-            url: '/preference/edition/au',
-            title: 'Australian edition',
-        },
-        {
-            url: '/preference/edition/int',
-            title: 'International edition',
-        },
-    ];
-
-    return (
-        <div className={editionDropdown}>
-            <Dropdown label="UK edition" links={links} id="edition" />
-        </div>
-    );
+const uk: Link = {
+    url: '/preference/edition/uk',
+    title: 'UK edition',
+    isActive: true,
 };
 
-export default EditionDropdown;
+const us: Link = {
+    url: '/preference/edition/us',
+    title: 'US edition',
+};
+
+const au: Link = {
+    url: '/preference/edition/au',
+    title: 'Australian edition',
+};
+
+const int: Link = {
+    url: '/preference/edition/int',
+    title: 'International edition',
+};
+
+const defaultEdition = 'UK';
```

Comment:
Could we move this variable declaration inside the scope of `EditionDropdown `'s constructor as it's not used outside of it?